src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.cs

```diff
@@ -1294,7 +1294,8 @@ private static T SerializeAndDeserialize<T>(T value, string baseline, DataContra
             if (!result.Equal && !skipStringCompare)
             {
                 Console.WriteLine(result.ErrorMessage);
-                throw new Exception(string.Format("Test failed for input : {0}", value));
+                Console.WriteLine(string.Format("Test failed for input : {0}", value));
+                Assert.StrictEqual(baseline, actualOutput);
```

Comment:
My intention is the later case. The baseline and actualOutput values here should be different. I wanted to print out the expected and actual values and Assert.StrictEqual is not a good choice. I will update to just use Console.WriteLine() for this purpose.